typescript.js

```diff
@@ -0,0 +1,408 @@
+/* eslint sort-keys: ["error", "asc"], quote-props: ["error", "consistent"] */
+/* eslint-disable sort-keys */
+
+module.exports = {
+    plugins: ["@typescript-eslint"],
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 2019,
+        sourceType: 'module',
+    },
+    plugins: [
+        'import',
+        'mocha',
```

Comment:
We don't have a plugin mocha installed you should add that too if you want to use it.

Reply:
@JoeTypeDigital the plugin mocha is still missing.